src/s3b_the_roach_market.tw2

```diff
@@ -344,3 +357,46 @@ I’d found this same box on this same table as a kid, and snatched it up immedi
 Honestly I was kinda hurt to see the box still on the table, but... whatever. Today’s ninjas had no fuckin’ taste.
 
 [[(back)|RM-junkheap-goodshit]]
+
+(set: $junkHeapItemsSeen to it +1)
+
+::RM-junkheap-flashback
+I walked up to the scooter and just took it in for a bit. It wasn't in good shape, nothing in this pile could say that.
+
+It was beaten to hell, with rust gathering all over it. The light had been busted out long ago by some ninja looking for a quick thrill. I felt myself pulled back to the [first time I saw the scooter]<scooterTrigger|.
+
+<div class="flashback">(click: ?scooterTrigger)[$dissolve[It’s a couple years ago and I’m waiting outside of the school trailer. I’m waiting for Anna, as usual.
```

Comment:
Repetitive use of "waiting." How about we replace the first use with "standing" or "hanging around"?

Reply:
Fixed